FunctionTesting/Tests/ro/sci/bv/CelsiusConverter/CelsiusConverterTest.java

```diff
@@ -0,0 +1,75 @@
+package ro.sci.bv.CelsiusConverter;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+
+/**
+ * This test checks if a Fahrenheit temperature is correctly transformed into Celsius degrees
+ * It also tests if the right message is given to the user, based on inputted values.
+ *
+ * @author Bogdan Butuza
+ */
+class CelsiusConverterTest {
+
+    CelsiusConverter c = new CelsiusConverter();
+
+    /**
+     * Testing conversion: 98.60 Fahrenheit degrees represent 37.00 Celsius degrees
+     */
+    @Test
+    public void convertCelsiusEquals() {
+        double expectedTemperature = c.convertCelsius(98.60);
+        Assertions.assertEquals(expectedTemperature, 37.00);
+    }
+
+    /**
+     * Testing conversion: -40 Fahrenheit degrees represent -40 Celsius degrees
+     * Not a valid human body temperature, but testing conversion
+     */
+    @Test
+    public void convertCelsiusEqualsBoth() {
+        double expectedTemperature = c.convertCelsius(-40);
+        Assertions.assertEquals(expectedTemperature,-40);
+    }
+
+    @Test
+    public void convertCelsiusNotEquals() {
+        double convert = c.convertCelsius(138.20);
+        //Expected result is 59 degrees
+        Assertions.assertNotEquals(convert, 80);
+    }
+
+    private final static String NORMAL_MESSAGE = "Your temperature is normal.";
+    private final static String ILL_MESSAGE = "You are ill!";
+    private final static String INVALID_MESSAGE = "The given temperature isn't for a human body.";
```

Comment:
as a convention, variables should be at the start of the class, not mixed between methods